microsoft-edge/devtools-protocol-chromium/index.md

```diff
@@ -191,7 +191,7 @@ None.
         "id": "2AE2506D9FDB1C541FB36DD908ED51DE",
         "title": "Remotely debug Windows devices - Microsoft Edge Development | Microsoft Learn",
         "type": "page",
-        "url": "https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/remote-debugging/windows",
+        "url": "https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/remote-debugging/windows",
```

Comment:
@captainbrosset - tech review this change

Reply:
Yes that's fine. This code snippet is just an example of what the API might return. Users will not need to use this particular URL. They'll use the API, and it will return a list of debuggable targets, each with their own URLs. So this is just to illustrate. It's fine to keep, or remove the `en-us` prefix here.

Reply:
removed en-us/